jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/SpringBootExplodedProcessor.java

```diff
@@ -41,20 +41,20 @@
 public class SpringBootExplodedProcessor implements JarProcessor {
 
   @Nullable private final Path jarPath;
-  @Nullable private final Path tempDirectoryPath;
+  @Nullable private final Path targetDirectoryPath;
 
-  public SpringBootExplodedProcessor(Path jarPath, Path tempDirectoryPath) {
+  public SpringBootExplodedProcessor(Path jarPath, Path targetDirectoryPath) {
```

Comment:
It's difficult to understand what `targetDirectoryPath` means. `targetExplodedJarRoot`? And maybe we can remove `localExplodedJarRoot` below.